sdks/js/packages/core/react/components/organization/plans/confirm-change/index.tsx

```diff
@@ -38,14 +38,14 @@ export default function ConfirmPlanChange() {
 
   const cancel = useCallback(() => navigate({ to: '/plans' }), [navigate]);
 
-  const expiryDate = useMemo(() => {
-    if (activePlan?.created_at && activePlan?.interval) {
-      return dayjs(activePlan?.created_at)
-        .add(1, activePlan?.interval as ManipulateType)
-        .format(config.dateFormat || DEFAULT_DATE_FORMAT);
-    }
-    return '';
-  }, [activePlan?.created_at, activePlan?.interval, config.dateFormat]);
+  // const expiryDate = useMemo(() => {
```

Comment:
Let's delete this?

Reply:
Discussed with @rsbh . This will be reinstated in future when we have a dry run API